packages/Angular/Explorer/explorer-core/src/lib/resource-browser/resource-browser.component.ts

```diff
@@ -271,89 +370,72 @@ export class ResourceBrowserComponent {
     this.ResourceSelected.emit(item);
   }
 
-  public backButtonClicked(){
```

Comment:
@JS-BC  did you remove my partial implementation of breadcrumbs?

Reply:
No, its called ` goToParent` in this component